docker/lookaside.go

```diff
@@ -30,6 +31,12 @@ const builtinRegistriesDirPath = "/etc/containers/registries.d"
 // userRegistriesDirPath is the path to the per user registries.d.
 var userRegistriesDir = filepath.FromSlash(".config/containers/registries.d")
 
+// defaultRootlessDockerDir is the default sigstore directory for per use configuration
+var defaultRootlessDockerDir = filepath.FromSlash(".local/share/containers/sigstore")
```

Comment:
`User`, `Nonroot`, `Unprivileged`, not `Rootless`, please; that means something more specific.